pythoncz/models/github.py

```diff
@@ -82,9 +82,9 @@ def _enhance_issue(session, issue):
     try:
         reactions = issue_details['reactions']
         issue['votes'] = (
-            reactions.get('total_count', 0)
-            - reactions.get('-1', 0)
-            - reactions.get('confused', 0)
+            reactions.get('total_count', 0) -
+            reactions.get('-1', 0) -
+            reactions.get('confused', 0)
```

Comment:
Shouldn't you rather update to a [newer linter](https://pypi.python.org/pypi/pycodestyle)?